test/wh_test_crypto.c

```diff
@@ -26,6 +26,7 @@
 #include <stdint.h>
 #include <stdio.h>  /* For printf */
 #include <string.h> /* For memset, memcpy */
+#include <unistd.h> /* For sleep */
```

Comment:
needs to be protected by `WOLFHSM_CFG_TEST_POSIX` (block already present later in this file)